src/platform-includes/distributed-tracing/custom-instrumentation/java.mdx

```diff
@@ -0,0 +1,66 @@
+On this page you will learn how to manually propagate trace information into and out of your Java application. Please note, that you do not need to do this manually, if you [use one of our supported frameworks](/platforms/java/usage/distributed-tracing/#how-to-use-distributed-tracing), or you have our <PlatformLink to="/performance/">performance monitoring feature</PlatformLink> turned on.
+
+To set it up manually, all you have to do is to make sure your application extracts incoming headers and to set those headers again when making an outgoing request within your application.
+
+## Step 1) Extract Incoming Tracing Information
+
+Incoming tracing information has to be extracted and stored in memory for later use. Sentry provides the `continueTrace()` function to help you with this. Incoming tracing information can come from different places:
+
+- In a web environment it will be sent with HTTP headers, for example, by another Sentry SDK used in your frontend project.
```

Comment:
Since for Java there is only one possible incoming way, we should not list only one option, but make it better readable as a sentence

Reply:
OK. Wasn't sure whether we were going to add other thinks like env var to Java SDK soon. Will change.